api/pkg/app/gorm_logger.go

```diff
@@ -69,7 +69,7 @@ func (w *prodWriter) Printf(format string, data ...interface{}) {
 				log = log.With("unknown", d)
 			}
 		default:
```

Comment:
can we remove default?

Reply:
why do we wanna remove it?

Reply:
because we are doing nothing in default atm